variable/technology/electricity-grid.yaml

```diff
@@ -1,25 +1,83 @@
 # List of variables related to the electricity grid
+# description of variables related to electricity demand
+ Cost of Demand-not-served|Electricity:
+   description: Cost of demand-not-served. Cost of load curtailment. Value of Lost Load (VoLL)
+   unit:  €/MWh
+
+Demand-not-served|Electricity:
+   description: Demand-not-served by spatial node
```

Comment:
Demand-not-served by spatial node
I would have written 
Demand-not-served per region
Would be more generic as the geographical zones or nodes... are defined as "regions"

Reply:
Ok, I got it.
It could be written as `Demand-not-server per region (i.e. aggregate-regions, countries, nuts1, nuts2, nuts3, or nodes).`

Reply:
Yes agree